unity/Assets/Plugins/Countly/CountlyManager.cs

```diff
@@ -195,9 +196,15 @@ protected void EndSession(long duration)
 
       builder.Append("&session_duration=");
       AppendConnectionData(builder, duration.ToString());
+			Log ("Requesting session end");
+		try {
+				WebRequest www = WebRequest.Create(appHost + "/i?" +builder.ToString());
```

Comment:
does this work properly on iOS/Android (with all stripping options enabled) for unity 3x and above? If not, how compatible is this?